lib/phoenix_live_view/html_formatter.ex

```diff
@@ -403,18 +414,12 @@ defmodule Phoenix.LiveView.HTMLFormatter do
   end
 
   defp to_tree([{:tag_close, name, _meta} | tokens], buffer, [{name, attrs, upper_buffer} | stack]) do
-    # Set format? as false in case the previous token is a text without whitespaces. So we can check
-    # for that when formatting inline elements.
-    format? =
-      case List.first(upper_buffer) do
-        {:text, text, _meta} ->
-          :binary.last(text) in '\s\t\r\n'
+    preserve? =
```

Comment:
@josevalim refactored it like that so it is easier to check when formatting. Without it, we wouldn't be able to pattern match in order to check for `contenteditable` and my goal was to keep them all using the same clause in the Algebra format. Let me know if you see it differently 🙌🏻